crates/aptos/src/account/multisig_account.rs

```diff
@@ -0,0 +1,244 @@
+        )?);
+        self.txn_options
+            .submit_transaction(aptos_stdlib::multisig_account_create_transaction(
+                self.multisig_account,
+                to_bytes(&payload).unwrap(),
+            ))
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+#[derive(Debug, Parser)]
+pub struct Approve {
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub(crate) multisig_account: AccountAddress,
+    #[clap(long)]
+    pub(crate) sequence_number: u64,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for Approve {
+    fn command_name(&self) -> &'static str {
+        "Approve"
+    }
+
+    async fn execute(self) -> CliTypedResult<TransactionSummary> {
+        self.txn_options
+            .submit_transaction(aptos_stdlib::multisig_account_approve_transaction(
+                self.multisig_account,
+                self.sequence_number,
+            ))
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+#[derive(Debug, Parser)]
+pub struct Reject {
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub(crate) multisig_account: AccountAddress,
+    #[clap(long)]
+    pub(crate) sequence_number: u64,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for Reject {
+    fn command_name(&self) -> &'static str {
+        "Reject"
+    }
+
+    async fn execute(self) -> CliTypedResult<TransactionSummary> {
+        self.txn_options
+            .submit_transaction(aptos_stdlib::multisig_account_reject_transaction(
+                self.multisig_account,
+                self.sequence_number,
+            ))
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+#[derive(Debug, Parser)]
+pub struct Execute {
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub(crate) multisig_account: AccountAddress,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for Execute {
+    fn command_name(&self) -> &'static str {
+        "Execute"
+    }
+
+    async fn execute(self) -> CliTypedResult<TransactionSummary> {
+        let payload = TransactionPayload::Multisig(Multisig {
+            multisig_address: self.multisig_account,
+            // TODO: Support passing an explicit payload
+            transaction_payload: None,
+        });
+        self.txn_options
+            .submit_transaction(payload)
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+#[derive(Debug, Parser)]
+pub struct ExecuteReject {
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub(crate) multisig_account: AccountAddress,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
```

Comment:
Each of these commands will need the following:
* A short description
* A longer description
* Documentation for each field
```
/// Rejects a multisig transaction
///
/// As a signer of multisig, rejects a transaction proposed by another multisig
/// signer. ...
#[derive(Debug, Parser)]
pub struct ExecuteReject {
    /// The multisig account address to sign
    ///
    /// This address should not be the same as the account signing
    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
    pub multisig_account: AccountAddress
    ...
}
```

Additionally, we should move the `multisig_account` to a common struct since it's used among all of these so it's documented once.

Reply:
Updated